Tests/Source/Model/UserClient/UserClientKeyStoreTests.swift

```
@@ -122,15 +120,15 @@ class UserClientKeysStoreTests: OtrBaseTest {
         
     }
     
-    private static func createFakeOTRFolder() {
-        try! NSFileManager.defaultManager().createDirectoryAtPath(UserClientKeysStore.legacyOtrDirectory.path!, withIntermediateDirectories: true, attributes: [:])
+    fileprivate static func createFakeOTRFolder() {
+        try! FileManager.default.createDirectory(atPath: UserClientKeysStore.legacyOtrDirectory.path, withIntermediateDirectories: true, attributes: [:])
     }
     
     func testThatTheNonEmptyLegacyOTRFolderIsDetected() {
         
         // given
-        self.dynamicType.createFakeOTRFolder()
-        "foo".dataUsingEncoding(NSUTF8StringEncoding)!.writeToURL(UserClientKeysStore.legacyOtrDirectory.URLByAppendingPathComponent("dummy.txt"), atomically: true)
+        type(of: self).createFakeOTRFolder()
+        try? "foo".data(using: String.Encoding.utf8)!.write(to: UserClientKeysStore.legacyOtrDirectory.appendingPathComponent("dummy.txt"), options: Data.WritingOptions.atomic)
```

Should we use `try!` instead? This way it will not silently fail.